presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestCountStatisticsBuilder.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata.statistics;
+
+import com.facebook.presto.common.block.Block;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.common.type.BigintType.BIGINT;
+import static org.testng.Assert.assertEquals;
+
+public class TestCountStatisticsBuilder
+{
+    @Test
+    public void testNoValues()
+    {
+        CountStatisticsBuilder statisticsBuilder = new CountStatisticsBuilder();
+        ColumnStatistics columnStatistics = statisticsBuilder.buildColumnStatistics();
+        assertEquals(columnStatistics.getNumberOfValues(), 0);
```

Comment:
Apparently, none of column writers report numberOfValues as Long, all writers use primitive boxing and hasNumberOfValues is always set to true.

```
public class ColumnStatistics implements Hashable
{
    private final boolean hasNumberOfValues;
    private final long numberOfValues;
    ...
    public ColumnStatistics(Long numberOfValues, HiveBloomFilter bloomFilter)
    {
        this.hasNumberOfValues = numberOfValues != null;
        this.numberOfValues = hasNumberOfValues ? numberOfValues : 0;
        ...
    }
```

Reply:
hasNumberOfValues can be removed, I was doing some clean up but did not finish it all the way through.